narwhal/primary/src/core.rs

```diff
@@ -318,10 +321,7 @@ impl Core {
                 .headers_suspended
                 .with_label_values(&[&header.epoch.to_string(), "missing_parents"])
                 .inc();
-            debug!(
-                "Processing of {} suspended: missing parent(s)",
-                header.digest()
-            );
+            debug!("Processing header {header} suspended: missing parent(s)");
```

Comment:
Could you please change the `{header}` to `{:?header}` instead? That will give us the debug version of print which is reacher

Reply:
To me the most interesting info is the round number and the digest which `Display` includes, and we have been using `Display` everywhere else. I think print header `Debug` at one place would be enough. If we don't have that, I can add it when the header is received.

Reply:
Actually `Dislay` prints round number and author. But I still think it is enough.